src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -24,7 +25,7 @@ const request = (opts) => {
   return $.ajax(ajaxOptions);
 };
 
-const Body = BaseForm.extend(Object.assign(
+const Body = BaseFormWithPolling.extend(Object.assign(
```

Comment:
Also, I don't think we need a baseForm just for polling.. BaseForm should be able to handle this as well..

Reply:
I originally had the listener in the `initialize()` function in BaseForm, but per @haishengwu-okta 's suggestion, that's not ideal because that logic is only applicable to certain views. Wasn't 100% sure where to put this logic exactly, so I'm open to any suggestions.

Reply:
Either leave to BaseForm or BaseFormWithPolling.
The reason I'd suggest `BaseFormWithPolling` is polling is only meant for several views and doesn't quite make sense to blow up BaseForm this way.

Reply:
With her new implementation we don't even need `BaseFormWithPolling` at all.. Concern view can look for change in value for `dynamicRefresh` attribute..

Reply:
Other view will need this soon, like email polling.

Reply:
```
import polling from './shared/polling';

const BaseFormWithPolling = BaseForm.extend({
   initialize() {
     // where can listen to dynamic refresh event
  },
...polling
});

----------

// using `polling` can be removed from `DeviceChallengePollView.Body` now.
DeviceChallengePollView.Body = BaseFormWithPolling.extend({...});

```

Potentially, all view (its body) using `polling` today can extend from BaseFormWithPolling.

There is general issue around inherent v.s. composition (cause no support multiple parent class) but we can worry it about later.

Reply:
In that case, I think @pradeepdewda-okta 's argument would be we can just listen for `change:dynamicRefresh` in the email polling view. I do think `BaseFormWithPolling` would help with redundancy

Reply:
I am good with this change, make sense to separate it out and create a reusable class. We can refactor other views later. Can we change the name to `BaseFormPolling.js`?